lld/test/ELF/Inputs/dwarf5-64-debug-names-type-comdat-helper.s

```diff
@@ -0,0 +1,397 @@
+	.text
```

Comment:
I imagine if you check the other test cases (for things like the DebugLoc/DebugRanges cases) you'll find they don't use full DWARF but probably just some relocations in an otherwise empty/small section - and we should probably test this in a similar way. (if the DebugLoc/DebugRanges tests do use fully formed DWARF, that'll be a surprise to me, but then happy to work with you to see if we can simplify the test(s) as much as possible in that form)

Reply:
Yeah I can look into simplifying.

Reply:
Simplified, or does it need to be even further that it's not valid .debug_names section, and just has reloc?

Reply:
I'd go further, not making it a valid .debug_names section - like the tests in debug-dead-reloc.s (or it could even be added to that - it tests comdat .text already, so it could test comdat .debug_types too) - that tests special cases for .debug_loc and .debug_ranges, for instance, but doesn't have whole valid loclists or range lists.

(but mostly whatever @maskray's happy with)